### PR TITLE
refactoring initialize

### PR DESCRIPTION
**Pull request by wesley974**
What do you think about this change ?

### CODE REVIEW

File: lib/scoreboard.rb [hunk 1]
```diff
@@ -5,8 +5,11 @@ class Scoreboard
   FLPATH = File.join(File.dirname('../'), 'score.yml')
   Row = Struct.new(:name, :tries, :time, :timestamp)
   def initialize
-    @score = YAML.load_file(FLPATH) if File.exist?(FLPATH)
-    @score = [] unless @score
+    @score = if File.exist?(FLPATH)
+               YAML.load_file(FLPATH)
+             else
+               []
+             end
```
Comment (kotp) [inline lib/scoreboard.rb]: `@score = File.exist?(FLPATH) && YAML.load_FILE(FLPATH) || []`

Just another way to express it.  Or you could also express it like this:

`@score = File.exist?(FLPATH) ? YAML.load_FILE(FLPATH) : []`

**Comment by kotp** [comment]
I really am not partial to the `FLPATH` constant name.  There may be a better name, but naming is hard.

The change is good though.

**Comment by wesley974** [comment]
Yours is far better. Well done !

**Comment by wesley974** [comment]
oups, my mistake

**Pull request merged**